src/components/admin/Billing/index.tsx

```diff
@@ -212,39 +204,41 @@ function AdminBilling() {
                     <CardWrapper
                         height={TOTAL_CARD_HEIGHT}
                         message={
-                            <FormattedMessage id="admin.billing.graph.dataByMonth.header" />
-                        }
-                    >
-                        <GraphStateWrapper>
-                            <DataByMonthGraph />
-                        </GraphStateWrapper>
-                    </CardWrapper>
-                </Grid>
-
-                <Grid item xs={12} md={6}>
-                    <CardWrapper
-                        height={TOTAL_CARD_HEIGHT}
-                        message={
-                            <FormattedMessage id="admin.billing.graph.taskHoursByMonth.header" />
+                            <FormattedMessage id="admin.billing.graph.usageByMonth.header" />
                         }
                     >
                         <GraphStateWrapper>
-                            <TaskHoursByMonthGraph />
+                            <UsageByMonthGraph />
                         </GraphStateWrapper>
                     </CardWrapper>
                 </Grid>
 
-                <Grid item xs={12} md={6}>
+                <Grid item xs={12} md={12}>
                     <CardWrapper
                         height={TOTAL_CARD_HEIGHT}
                         message={
-                            <FormattedMessage id="admin.billing.graph.dataByTask.header" />
+                            <>
+                                <FormattedMessage id="admin.billing.label.lineItems" />
+
+                                {selectedInvoice ? (
+                                    <DateRange
+                                        start_date={selectedInvoice.date_start}
+                                        end_date={selectedInvoice.date_end}
+                                    />
+                                ) : (
+                                    ' ...'
```

Comment:
Just thinking... this might be a good spot for a `circularProgress` component or `skeleton` so that it is clear something is loading in here.